component-samples/demo/README.md

```diff
@@ -113,7 +113,7 @@ Note that this requires internet access by the component. Should a component be
 To enable OnDie support in FDO PRI Manufacturer and FDO PRI Owner, update manufacturer.env and owner.env respectively as below.
 
 ```
-ondie_cache=file:///home/manufacturer/ # file:///home/owner for FDO PRI Owner
+ondie_cache=file:///home/manufacturer/<ondie-folder>/ # file:///home/owner/<ondie-folder>/ for FDO PRI Owner
```

Comment:
This snippet is expected to be copied as is. Replace '<ondie-folder>' with 'ondie_cache_dir' or equivalent.